packages/upload/src/vaadin-upload.js

```diff
@@ -709,7 +703,6 @@ class Upload extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElement))
         clearTimeout(stalledId);
         file.indeterminate = file.uploading = false;
         if (file.abort) {
-          this._updateFileList();
```

Comment:
This shouldn't be needed anymore after #4894 as there is no need to re-render file list.